packages/components/src/DataViewer/RecordsViewer/Branch/RecordsViewerBranch.component.js

```diff
@@ -110,7 +109,7 @@ export class RecordsViewerBranch extends React.Component {
 						onToggle={this.props.onToggle}
 						opened={opened}
 						style={{
-							marginLeft: type === 'object' ? ICON_MARGIN_PLUS : ICON_MARGIN_CARET,
+							marginLeft: ICON_MARGIN,
```

Comment:
Is it possible to not place the icon the way it is ? :) 
Let's all do that in a simple sass file
```css
.tc-records-viewer-branch-content {
    position: relative;
}

.tc-tree-branch-icon {
    position: absolute;
    left: 0;
    top: 50%;
    transform: translate(-150%, -50%);
}
```

And remove the js style that is now constant.

Plus, I know it's not in your scope but this button is not accessible. :( There is no text alt.